### DISCUSSION TITLE
Is it possible to use a shared GitHub Enterprise default.json in Bitbucket?

### DISCUSSION THREAD

**Discussion by Keysox** [renovatebot/renovate#16072] [Request Help]
We tried a few different syntaxes to extend from based on the [documentation](https://docs.renovatebot.com/config-presets/) but couldn't seem to get this working. We were hoping for something such as `"extends": ["github@privatedomain.com>org/repo"]`

https://github.com/renovatebot/renovate/blob/6a784366009aa1bee724012f016e3c3c6f0ce365/lib/config/presets/github/index.ts#L9 
It looks like it may be hard-coded in this file but it does appear to work when going from GitHub Enterprise -> GitHub Enterprise, which I assume is using the `local` option but maybe there's something I'm missing!

**Comment by viceice** [comment]
not possible, only public cloud hosted or local presets are supported.

**Reply by Keysox** [reply]
Awesome, thanks for the info! Does local preset mean same domain or is there a way to share a config between GitHub Enterprise and Bitbucket using that?

**Reply by viceice** [reply]
you can only use GitHub.com, gitlab.com or gitea.io as public shared preset hosting

**Reply by Keysox** [reply]
Good to know. Would it be alright if I created an issue to add the feature we're looking for?

**Reply by viceice** [reply]
only if you like to support bitbucket cloud. we don't like to support shared presets for cross self-hosted platforms.

**Reply by Keysox** [reply]
Bummer, thanks for your help
